<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FingerGuessing</class>
 <widget class="QDialog" name="FingerGuessing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FingerGuessing</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="stoneButton">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="text">
       <string>石头</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="scissorsButton">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="text">
       <string>剪子</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clothButton">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="text">
       <string>布</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
